pdata/plog/pb.go

```diff
@@ -0,0 +1,59 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plog // import "go.opentelemetry.io/collector/pdata/plog"
+
+import (
+	"go.opentelemetry.io/collector/pdata/internal"
+	otlplogs "go.opentelemetry.io/collector/pdata/internal/data/protogen/logs/v1"
+)
+
+// NewProtobufLogsMarshaler returns a LogsMarshaler. Marshals to OTLP binary protobuf bytes.
+func NewProtobufLogsMarshaler() LogsMarshaler {
```

Comment:
I think we should shortcut to just `Proto` instead of `Protobuf` or update the `plogotlp.LogsRequest` to use `MarshalProtobuf`.

Reply:
Yeah, good catch. With all the discussions above this signature would become:

```golang
func NewProtoMarshaler() Marshaler {
```